draft-ietf-tls-tls13.md

```diff
@@ -3552,7 +3582,8 @@ The following rules apply to the certificates sent by the server:
 
 - The server's end-entity certificate's public key (and associated
   restrictions) MUST be compatible with the selected authentication
-  algorithm (currently RSA, ECDSA, or EdDSA).
+  algorithm from the client's "signature_algorithms" extension
```

Comment:
This seems unnecessary.

Reply:
The idea was to clarify that signature_algorithms_cert is not relevant for this choice, as opposed to in the next bullet point.

Reply:
OK, I can live with it.